src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/Runners/MonitorCollectRunner.CollectionRule.cs

```diff
@@ -38,19 +38,57 @@ public async Task WaitForCollectionRuleCompleteAsync(string ruleName, Cancellati
             }
         }
 
+        public async Task WaitForCollectionRuleUnmatchedFiltersAsync(string ruleName, CancellationToken token)
```

Comment:
Just noticing the code reuse; should we have these two methods just call a common method that takes the `LoggingEventId` as a parameter? Just seems like something we'd have to update twice since it looks like everything else in these methods is identical.